lib/modules/onc_program/bulk_data_export_sequence.rb

```diff
@@ -39,6 +39,7 @@ def check_capability_statement
         headers = { accept: 'application/fhir+json' }
         reply = LoggedRestClient.get(url, headers)
```

Comment:
We should use the fhir_client here and the built in capabilities functionality at this point, I think.

Reply:
fixed